src/flowgger/encoder/rfc5424_encoder.rs

```diff
@@ -92,12 +97,14 @@ impl Encoder for RFC5424Encoder {
 use crate::flowgger::record::{SDValue, StructuredData};
 #[cfg(test)]
 use crate::flowgger::utils::test_utils::rfc_test_utils::ts_from_date_time;
+#[cfg(test)]
+use time::Month;
 
 #[test]
 fn test_rfc5424_encode() {
-    let expected_msg = r#"<13>1 2015-08-06T11:15:24.637Z testhostname - - - some test message"#;
+    let expected_msg = r#"<13>1 2015-08-06T11:15:24.638Z testhostname - - - some test message"#;
```

Comment:
was there any reason for changing 637 to 638 ? not that it matters but i'm curious

Reply:
There is some rounding issue with that specific number in Rust:

```
fn main() {
    let mut n: f64;

    n = 1438859724636000000.0 as f64;
    println!("n: {}", n / 1e9);

    n = 1438859724637000000.0 as f64;
    println!("n: {}", n / 1e9);
    
    n = 1438859724638000000.0 as f64;
    println!("n: {}", n / 1e9);    
}
```

Running this gives us:

```
n: 1438859724.636
n: 1438859724.6369998
n: 1438859724.638
```

Reply:
I don't see any concerns with this as this would mean 1 ms up or down